framework/include/mesh/FileMesh.h

```diff
@@ -28,14 +29,13 @@ class FileMesh : public MooseMesh
 public:
   FileMesh(const InputParameters & parameters);
   FileMesh(const FileMesh & other_mesh);
-  virtual ~FileMesh();
```

Comment:
This is restored but your diff still appears here because I added a comment to this line. Rest assured, it is fixed.